app/views/collections/show.html.erb

```diff
@@ -6,7 +6,7 @@
 
 <div id="content" class="col-md-9 col-sm-8">
   <div itemscope itemtype="http://schema.org/CollectionPage" class="row">
-    <div class="col-sm-10 pull-right">
```

Comment:
this cause the image to display on the right side of the screen at large resolutions.  is that intentional?